README.md

```diff
@@ -46,8 +46,10 @@ For instance:
 ```js
 const { fastifyRequestContextPlugin } = require('fastify-request-context')
 const fastify = require('fastify');
+const middie = require('middie')
 
 const app = fastify({ logger: true })
+app.register(middie)
```

Comment:
Is this needed?

Reply:
Will try removing it, I thought middie is mandatory now for using plugins?..

Reply:
Removed it, test still passes.

Reply:
middie is not mandatory. It's mandatory only if you want to use `.use()`, e.g. express-style middlewares.

Reply:
is it adding chaos in the readme then?